cupyx/scipy/ndimage/filters.py

```diff
@@ -999,11 +999,47 @@ def _get_rank_kernel(filter_size, rank, mode, w_shape, offsets, cval,
     # Also tried insertion sort, which is always slower than either one
     sorter = __SELECTION_SORT if filter_size <= 255 else \
```

Comment:
It looks like the new code below will get called rather than _SELECTION_SORT when filter_size <=255, so should _SELECTION_SORT just be removed?

Reply:
Yes, this PR implementation is always faster than the general selection sort implementation, so maybe we should remove it.